web/beacon-app/src/components/common/Modal/ApiKeyModal/ApiKeyModal.tsx

```diff
@@ -79,21 +89,51 @@ export default function ApiKeyModal({ open, onClose, data }: ApiKeyModalProps) {
                 </div>
               </div>
               <div className="absolute top-3 right-3 flex gap-2">
-                <button onClick={() => handleDownload(clientInfo, 'client')} data-testid="download">
+                <button
+                  onClick={() => handleDownload(clientInfo, `APIKey-${data?.name}-${dateCreated}`)}
```

Comment:
Updating the API key filename to match a change made in a previous PR.

Reply:
Does the downloaded file name include the project name? If not, I think it would be extremely helpful to include that as part of the file name.

Reply:
It currently doesn't, but I can add the project name.